pkgs/development/python-modules/python-redis-lock/default.nix

```
@@ -30,7 +31,7 @@ buildPythonPackage rec {
   disabledTests = [
     # https://github.com/ionelmc/python-redis-lock/issues/86
     "test_no_overlap2"
-  ];
+  ] ++ lib.optionals stdenv.isDarwin [ "test_reset_signalizes" "test_reset_all_signalizes" ];
```

```suggestion
  ] ++ lib.optionals stdenv.isDarwin [
    "test_reset_signalizes"
    "test_reset_all_signalizes"
  ];
```
Why do these tests fail?

---

Fixed. Thanks!